_layouts/card.html

```diff
@@ -1,25 +1,14 @@
 <!DOCTYPE html>
 <html lang='en'>
-{% include head.html %}
-
-<body class="layout--card">
-  {% include category-setup.html %}
-  {% include header.html %}
-  <main id="main" role="main">
-    <section class="category category--{{page.category | downcase }} usa-section" id="{{page.category | downcase }}">
-    <header class="category--header">
-        <div class="usa-grid">
-            {% assign method_category = site.pages | where: "title", page.category | first %}
-            <h1 class="category--title">{{method_category.title | capitalize }}</h1>
-            <p class="category--subtitle">{{method_category.description}}</p>
-        </div>
-    </header>
-    <div class="usa-grid">
-      {% include method-print-header.html %}
-      {% include method.html %}
-    </div>
-  </section>
-</main>
-{% include footer.html %}
-</body>
```

Comment:
I think this and other visual updates should be shelved until we get through the current round of user interviews.